src/internal.c

```diff
@@ -17820,6 +17820,28 @@ int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
         return ret;
     }
 
+#ifdef HAVE_SERVER_RENEGOTIATION_INFO
+
+    /* search suites for specific one, idx on success, negative on error */
+    static int FindSuite(Suites* suites, byte first, byte second)
+    {
+        int i;
+
+        if (suites == NULL) {
```

Comment:
Needs an || check for suiteSz == 0 as well.  Because if suiteSz is 0 then suiteSz - 1 on 17,835 will be a big number (word16).